packages/api/lambdas/sf-event-sqs-to-db-records/write-execution.js

```diff
@@ -130,6 +130,13 @@ const writeExecutionToDynamoAndES = async (params) => {
   }
 };
 
+const publishExecutionSnsMessage = async (record) => {
+  const topicArn = process.env.execution_sns_topic_arn;
+  const message = DynamoDB.Converter.marshall(record);
```

Comment:
So we don't need this. This function would be converting a Javascript object into a DynamoDB style record, as per https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/Converter.html#marshall-property. 

However, what Cloud metrics is expecting is a regular Javascript object representation of a record. 

That's why we're calling `unwrap` in the existing `publish-executions` code: to unwrap a DynamoDB style object into a JS object:

https://github.com/nasa/cumulus/blob/master/packages/api/lambdas/publish-executions.js#L17

You can also see that this is the expected behavior from the unit tests:

https://github.com/nasa/cumulus/blob/master/packages/api/tests/lambdas/test-publish-executions.js#L41